package.json

```diff
@@ -45,7 +45,7 @@
     "broccoli-sourcemap-concat": "^0.4.3",
     "broccoli-string-replace": "0.0.2",
     "git-repo-version": "^0.1.1",
-    "testem": "^0.7.6",
+    "testem": "git://github.com/testem/testem.git#a63d877",
```

Comment:
You should be able to specify `^1.0.0-rc.1` to grab this from `npm` instead of GH.  Also, will allow grabbing 1.0.0 when it is released...